lib/Charts/LineChart.js

```diff
@@ -287,16 +296,18 @@ function render(element, state) {
     if (state.mini) {
```

Comment:
I realise the transition on line 291 may have a different duration - but this syntax looks the same as in v3 - should you use the `d3Transition()` syntax here too?